build/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.5
+FROM golang:1.5.1
```

Comment:
Even if the `golang:1.5` tag has been updated, if we don't update the dockerfile then anyone who rebuilds the builder image without having pulled recently could recreate it with the old version. Is there any way to specify a minimum version in the 1.5 series? (or add `docker pull` to the build script?)